site/src/pages/DeploymentSettingsPage/GeneralSettingsPage/GeneralSettingsPageView.tsx

```diff
@@ -54,6 +61,37 @@ export const GeneralSettingsPageView: FC<GeneralSettingsPageViewProps> = ({
 						</ChartSection>
 					</div>
 				)}
+				{licenseUtilizationPercentage && (
+					<ChartSection title="License Utilization">
+						<LinearProgress
+							variant="determinate"
+							value={(licenseUtilizationPercentage % 1) * 100}
```

Comment:
```suggestion
							value={Math.min(licenseUtilizationPercentage * 100, 100)}
```

as is, if the percentage was 105% we would show it as 5%. I think this is what you meant to do. :)